libflux/flux-core/src/semantic/sub.rs

```diff
@@ -1,19 +1,18 @@
-#![allow(missing_docs)]
+//! Substitutions during type inference.
 
 use crate::semantic::types::{MonoType, SubstitutionMap, Tvar};
 
-// A substitution defines a function that takes a monotype as input
-// and returns a monotype as output. The output type is interpreted
-// as being equivalent to the input type.
-//
-// Substitutions are idempotent. Given a substitution s and an input
-// type x, we have s(s(x)) = s(x).
-//
+/// A substitution defines a function that takes a monotype as input
+/// and returns a monotype as output. The output type is interpreted
+/// as being equivalent to the input type.
+///
+/// Substitutions are idempotent. Given a substitution *s* and an input
+/// type *x*, we have *s*(*s*(*x*)) = *s*(*x*).
```

Comment:
These italics are mathematically inspired style. We could use backticks for pseudo-code style instead, if preferred (`s(s(x)) = s(x)`).